_episodes_rmd/04-making-packages-R.Rmd

```diff
@@ -6,14 +6,17 @@ questions:
 - "How do I collect my code together so I can reuse it and share it?"
 - "How do I make my own package(s)?"
 - "How do I integrate my function documentation into R's help pages system?"
+- "What helper tools does RStudio provide me with?"
 objectives:
 - "Describe the required structure of R packages."
 - "Create the required structure of a simple R package."
-- "Using roxygen2 to convert function docu into help pages."
+- "Build and install the package in RStudio."
+- "Use roxygen2 to convert function docu into help pages."
```

Comment:
docu -> document

Reply:
Unless there is a reason it says "docu"... not sure why though. It seems that this docu word is used throughout.